Разметка

SerialEditor opened the pull request:
---
:mortar_board: [Разметка](https://up.htmlacademy.ru/adaptive-individual/1/user/1745893/tasks/3)

:boom: https://htmlacademy-adaptive.github.io/1745893-cat-energy-1/2/

SerialEditor committed:
Добавляет страницы проекта

Добавляет страницы каталога продукции и формы подбора программы питания;
заполняет теги title на всех страницах

SerialEditor committed:
Добавляет контентную графику всех страниц

SerialEditor committed:
Заменяет файлы проекта

Заменяет фото товаров в каталоге на другие версии

SerialEditor committed:
Добавляет в проект логотип академии

SerialEditor committed:
Добавляет разметку и файлы

Добавляет разметку хедера и футера на страницы сайта, подключает
контентные изображения в размеченных разделах, добавляет файлы
иконок соцсетей в проект

SerialEditor committed:
Добавляет раздел main на страницы сайта

SerialEditor committed:
Добавляет разметку страниц

SerialEditor committed:
Добавляет вёрстку страницы каталога

SerialEditor committed:
Добавляет разметку формы анкетирования

SerialEditor committed:
Дополняет разметку страниц сайта

Добавляет необходимые атрибуты полей формы, добавляет альт-тексты

SerialEditor committed:
Исправляет разметку

SerialEditor commented on source/index.html:
```diff
@@ -62,18 +62,18 @@ <h2 class="features__title">Как это работает</h2>
       <section class="result">
         <h2 class="result__title">Живой пример</h2>
         <p class="result__description">Борис сбросил 5 кг за 2 месяца, просто заменив свой обычный корм на Cat Energy Slim. Отличный результат без изнуряющих тренировок! При этом он не менял своих привычек и по-прежнему спит по 16 часов в день.</p>
-        <dl class="result__stats metrics">
```
Андрей, генератор бэм-дерева декларирует проблему на уровне элементов блока metric, не могу разобраться, в чём проблема

andreysgra replied:
Павел, у БЭМ блока `metric` вместо латинской `c` русская буква. У элементов буква правильная.

andreysgra commented on source/form.html:
```diff
@@ -0,0 +1,147 @@
+          </fieldset>
+          <div class="customer-profile__group">
+            <label class="customer-profile__title" for="cat-habits-comment">Комментарий</label>
+            <textarea class="customer-profile__input input" name="cat-habits"   id="cat-habits-comment">Расскажите обо всех повадках кота</textarea>
```
"Расскажите обо всех повадках кота" - а это не `placeholder`?

SerialEditor replied:
Э-х-х... Это именно он!

andreysgra commented on source/catalog.html:
```diff
@@ -0,0 +1,270 @@
+        <img src="./img/content/common/header-logo-text.svg" width="101" height="18" alt="CAT ENERGY.">
+      </a>
+      <nav class="main-header__nav main-nav">
+        <button class="main-nav__toggle nav-menu-toggle">
```
Тип кнопки не указан.

andreysgra commented on source/index.html:
```diff
@@ -1,12 +1,137 @@
+        <p class="result__description">Борис сбросил 5 кг за 2 месяца, просто заменив свой обычный корм на Cat Energy Slim. Отличный результат без изнуряющих тренировок! При этом он не менял своих привычек и по-прежнему спит по 16 часов в день.</p>
+        <dl class="result__stats metrics-list">
+          <div class="metrics-list__inner-wrapper metriс">
+            <dt class="metric__description">снижение веса</dt>
+            <dd class="metric__value">5 кг</dd>
+          </div>
+          <div class="metrics-list__inner-wrapper metriс">
+            <dt class="metric__description">затрачено времени</dt>
+            <dd class="metric__value">60 дней</dd>
+          </div>
+          <div class="metrics-list__inner-wrapper metriс">
+            <dt class="metric__description">Затраты на питание:</dt>
+            <dd class="metric__value">15 000 руб.</dd>
```
![image](https://github.com/htmlacademy-adaptive/1745893-cat-energy-1/assets/17790771/4a44719d-8dc6-4490-b01f-7b9ce929d84e)
В `<div class="metrics-list__inner-wrapper metriс">` с буквой `c` в названии класса `metriс` проблемы, использована русская буква вместо латинской.

SerialEditor replied:
![metric_section](https://github.com/htmlacademy-adaptive/1745893-cat-energy-1/assets/81110169/669feae2-c8e3-4169-9df1-c9c37dc8d1cc)
Только после того, как поменял клавиатуру)

andreysgra reviewed (changes requested):
Кроме перечисленного ниже всё выполнено отлично.

SerialEditor committed:
Исправляет разметку

Добавляет тип button для кнопки меню навигации; добавляет плейсхолдер
полю ввода комментария; исправляет именование классов в секции result
